src/views/blog/admin/path_list.vue

```diff
@@ -165,86 +361,273 @@ export default class PathList extends Vue {
     this.paths = {};
     AdminAuth.attempt_get_JWT()
       .then((token) => {
-        api(this.client)
-          .blog.articles.$get()
-          .then((data: BlogArticle[]) => {
-            for (const article of data) {
-              this.$set(this.paths, article.id, {
-                category: article.category,
-                title: article.title,
-                created_at: article.created_at,
-                updated_at: article.updated_at,
-                waiting_count: 0,
-              });
-            }
-          });
-        return token;
+        // get articles
+        return new Promise<JWT>((resolve) =>
+          api(this.client)
+            .blog.articles.$get()
+            .then((data: BlogArticle[]) => {
+              for (const article of data) {
+                this.$set(this.paths, article.id, {
+                  category: article.category,
+                  title: article.title,
+                  author: article.author,
+                  handle_name: article.handle_name,
+                  created_at: article.created_at,
+                  updated_at: article.updated_at,
+                  waiting_count: 0,
+                  revision_id: article.revision_id,
+                  article_exists: true,
+                  revisions: [],
+                  revision_selection: article.revision_id,
+                  category_selection: article.category,
```

Comment:
spreadはいかがですか
エディタ開いてないから余計な要素があるのかもしれん 熟考してください(丸投げ)
```suggestion
                  ...article,
                  waiting_count: 0,
                  article_exists: true,
                  revisions: [],
```